code-review/README.md

```diff
@@ -65,6 +65,10 @@ experience, refactors the existing code). Then:
 - Seek to understand the author's perspective.
 - Sign off on the pull request with a 👍 or "Ready to merge" comment.
 - Remember that you are here to provide feedback, not to be a gatekeeper.
+- When suggesting changes:
```

Comment:
Is it correct that this advice pertains specifically to the "Add a suggestion" feature? If so, I think it might be helpful to say that explicitly. Otherwise I think people might interpret this as being for any type of suggestion, and not necessarily just this particular situation. What do you think?

```suggestion
- When suggesting changes using the "Add a suggestion" feature:
```

Reply:
Totally agree, thank you @jutonz